cherrypy/tutorial/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM python:3-alpine
+
+COPY . /tutorial
+
+WORKDIR /tutorial
+
+RUN pip install cherrypy && \
+  sed -i 's/^server\.socket_host.*$/server.\socket_host = "0.0.0.0"/' \
+    tutorial.conf
+
+
+ENTRYPOINT ["python"]
+
+CMD ["tut01_helloworld.py"]
```

I just realized that I prefer `python -m` invocation style.
```suggestion
CMD ["tut01_helloworld"]
```